beacon_node/network/src/beacon_processor/worker/rpc_methods.rs

```diff
@@ -675,13 +739,35 @@ impl<T: BeaconChainTypes> Worker<T> {
                     );
                     break;
                 }
+                Err(BeaconChainError::BlobsUnavailable) => {
```

Comment:
We could log different errors in the case where blob was empty vs when we didn't have the blob in our db. The first case isn't an issue but the second case is.